feature_engine/selection/drop_features.py

```diff
@@ -5,6 +5,9 @@
 from feature_engine.dataframe_checks import check_X
 from feature_engine.selection.base_selector import BaseSelector
 from feature_engine.tags import _return_tags
+from feature_engine._variable_handling.variable_type_selection import (
```

Comment:
Could we also run isort to order the imports?

`isort path/to/file`

or alternatively order the imports manually. The last one starts with _, so it should be at the top of the feature engine imports more or less.

Reply:
alright, done.